components/shared/Auth.vue

```diff
@@ -30,29 +33,45 @@ const components = {
 export default class Auth extends Vue {
   @Prop({ default: 24 }) public size!: number
   private balance = ''
+  private interval
 
   public mounted() {
     if (this.account) {
       this.$emit('input', this.account)
-      // this.calculateBalance(this.account);
+      this.calculateBalance(this.account)
+      this.pollBalance(this.account)
     }
   }
 
   set account(account: string) {
-    console.log('setAuth', account)
     this.$store.dispatch('setAuth', { address: account })
-    // this.calculateBalance(account);
+    this.calculateBalance(account)
+    this.pollBalance(account)
   }
 
   get account() {
     return this.$store.getters.getAuthAddress
   }
 
   protected async calculateBalance(account: string) {
-    console.log('calling balance', account)
     const { api } = Connector.getInstance()
     const balance = await api?.query.system.account(account)
     this.balance = balance?.data.free.toString()
+    this.$emit('update-balance', this.balance)
+  }
+
+  private async pollBalance(account) {
+    if (this.interval) {
+      clearInterval(this.interval)
+    }
+
+    this.interval = setInterval(() => {
+      this.calculateBalance(account)
+    }, 5000)
```

Comment:
Why would you need this interval?

Reply:
probably if has on one tab opened and meanwhile balanced has changed on the second tab? Hence our app doesn't communicate between tabs :|

Reply:
outdated